tests/integration/IbexaTestKernel.php

```diff
@@ -72,11 +72,19 @@ protected static function getExposedServicesById(): iterable
         yield 'ibexa.cache_pool' => TagAwareAdapterInterface::class;
     }
 
+    protected function skipOverridingCustomTagConfig(): bool
+    {
+        return false;
+    }
+
     public function registerContainerConfiguration(LoaderInterface $loader): void
     {
         parent::registerContainerConfiguration($loader);
 
-        $loader->load(__DIR__ . '/../lib/_settings/common.yaml');
+        if (!$this->skipOverridingCustomTagConfig()) {
+            $loader->load(__DIR__ . '/../lib/_settings/common.yaml');
+        }
```

Comment:
We don't need to complicate this like that. This config was meant for Legacy test setup and is loaded [elsewhere](https://github.com/ibexa/fieldtype-richtext/blob/6bb513074b5c37594d3d4fe3713ce8d13927c7b1/tests/integration/Repository/SetupFactory/RichTextSetupFactoryTrait.php#L47). Moreover they were added there, because at the time they were created, there were no built-in custom tags.  

IbexaTestKernel-based tests don't actually need those, they just need resource paths overrides.

I'd suggest:

1. dropping changes to the kernel and custom kernel for transaltions
2. creating `tests/integration/Resources/override.yaml` with resources override only
3. loading the created config instead of the one from Legacy setup

This way both legacy & current test setup should work and not affect translation test (as custom tags are not overridden).

I've summarized those changes in #163. Let's see if CI agrees.